Doc/c-api/typeobj.rst

```diff
@@ -2090,6 +2092,15 @@ and :c:type:`PyType_Type` effectively act as defaults.)
    .. versionadded:: 3.9 (the field exists since 3.8 but it's only used since 3.9)
 
 
+.. c:member:: char PyTypeObject.tp_watched
```

Comment:
I would rather the docs say "Internal. Do not use."

TBH, I would like the whole of the type struct to be "do not use", but let's not make things any worse.

Reply:
Are you wanting the existing text removed and replaced so it says _only_ "Internal, do not use"? Or do you want to add "Internal, do not use" but also keep the description of what it is?

Reply:
Pushed a version that removes the existing text, but I can easily re-add it.